instance/models/mixins/ansible.py

```diff
@@ -35,15 +36,31 @@
 
 # Classes #####################################################################
 
+# Represents and ansible playbook. It can be located in a remote git repository or
```

Comment:
Nit: spelling check - should be `Represents an ansible playbook`.

Reply:
Thanks for catching that; I fixed it.